test/integration/DCAHubSwapper/single-pair-swap-with-dex.spec.ts

```diff
@@ -1,262 +0,0 @@
-import { expect } from 'chai';
```

Comment:
Needed to reduce the amount of tests so we didn't get rate-limited, and since we already have the multi-pair-swap test, decided to delete this one